src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -54,14 +65,106 @@ public Task<bool> IsEnabledAsync<TContext>(string feature, TContext appContext,
 
         public async IAsyncEnumerable<string> GetFeatureNamesAsync([EnumeratorCancellation] CancellationToken cancellationToken)
         {
-            await foreach (FeatureDefinition featureDefintion in _featureDefinitionProvider
-                                .GetAllFeatureDefinitionsAsync(cancellationToken)
-                                .ConfigureAwait(false))
+            await foreach (FeatureDefinition featureDefintion in _featureDefinitionProvider.GetAllFeatureDefinitionsAsync(cancellationToken).ConfigureAwait(false))
             {
                 yield return featureDefintion.Name;
             }
         }
 
+        public ValueTask<T> GetVariantAsync<T, TContext>(string feature, TContext appContext, CancellationToken cancellationToken)
```

Comment:
Is null `appContext` considered a valid case?

Reply:
Yes, null is valid here as well as with contextual feature filters.

Reply:
Okay, then it makes sense not having a null check.